Django/robotechnics/hardathon/urls.py

```diff
@@ -1,16 +1,14 @@
-from django.urls import path, include
-from hardathon.router import (hardatons_router,
-                              detail_project_router,
-                              projects_router,
-                              partners_router,
-                              detail_hardaton_router)
+from django.urls import path
+from hardathon.router import (HardathonViewSet,
+                              HardatonProjectsViewSet,
+                              HardatonPartnersViewSet)
```

Comment:
Здесь та же ситуация с неймингом

```suggestion
from hardathon.router import (HardathonViewSet,
                              HardathonProjectsViewSet,
                              HardathonPartnersViewSet)
```